core/Archive.php

```diff
@@ -319,6 +320,8 @@ public function getNumeric($names)
      *                                to the subtable ID to return. If set to 'all', all subtables
      *                                of each requested report are returned.
      * @return array An array of appropriately indexed blob data.
+     *
+     * @deprecated since Piwik 2.12. Use one of the getDatable* methods instead.
      */
```

Comment:
Is there a reason for deprecating `getBlob()`. I agree it doesn't really serve a use nor is it a good idea to be exposed as an `@api` method since it's not backend agnostic, but perhaps it would be better to deprecate it in a separate pull request. Since it's not really related to this one.